src/seidr.py

```diff
@@ -90,7 +97,7 @@ def plot(self):
         legend.get_frame().set_alpha(0.5)
         for spine in ('top', 'right', 'bottom', 'left'):
             ax.spines[spine].set_visible(False)
-        plt.savefig(f"{OUTPUT_DIRECTORY_PATH}/S{self.S0}_E{self.E0}_I{self.I0}_D{self.D0}_R{self.R0}_B{self.beta}_a{self.alpha}_g{self.gamma}_m{self.mi}_l{self.lambda_}_e{self.epsilon}.png")
```

Comment:
It occured to me that we should probably have the "signature" of these files be a seperate (private) method.